opencensus/pom.xml

```diff
@@ -15,7 +15,9 @@
  limitations under the License.
 -->
 <!-- [START opencensus_pom] -->
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Can we intend this in alignment with other files?

Reply:
Fixed it!